website/blog/2019-02-20-single-spa-inspector.md

```diff
@@ -0,0 +1,39 @@
+---
+title: single-spa Inspector and 4.1
+author: Anthony Frehner
+authorURL: https://twitter.com/aahfrena
+authorImageURL: https://pbs.twimg.com/profile_images/1037162092963348480/1XLEV4MK_400x400.jpg
+---
+
+## Background
+For a long time, Canopy has had the benefit of using a tool called [sofe inspector](https://github.com/CanopyTax/sofe-inspector) (note that this is a really old version) to help do some things single-spa applications. There has always been a desire to [figure out how to share this tool so others can benefit as well](https://github.com/CanopyTax/single-spa/issues/151).
```

Comment:
`to help do some things single-spa applications.`
?
to help visualize, override, and maintain single-spa applications in the browser.

might be worth elaborating on what exactly Canopy uses its inspector for since that isn't immediately clear but might be useful in explaining the motivation behind single-spa-inspector